processor/src/main/java/com/epam/learn/processor/format/SongLengthSerializer.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.learn.processor.format;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+
+public class SongLengthSerializer extends JsonSerializer<Long> {
+
+    private static final String SONG_LENGTH_FORMAT = "%02d:%02d";
```

Comment:
There is no reason to keep it static as it is used only once.

Reply:
Fixed